### ISSUE TITLE
when i run the command "make", it entailed the error.

### ISSUE THREAD

**Issue by fantasticboy19** [paul-lee-attorney/fabric-2.1-gm#1]
![issue1](https://user-images.githubusercontent.com/31614496/101984252-14c5f280-3cbb-11eb-8c96-cc6fe8a59f72.png)

**Comment by fantasticboy19** [comment]
if you can not see the pic, the error info is etcdraft/store.go:19:2: module go.etcd.io/etcd@latest found (v3.3.25+incompatible), but does not contain package go.etcd.io/etcd/etcdserver/api/snap

**Comment by paul-lee-attorney** [comment]
Yes, This error will also incurr when build the src.  It seems like the package go.etcd.io/etcd does not support the snap func for server.Regards.PaulFrom My Huawei Mate20-------- 原始邮件 --------发件人： immortal-fossil <notifications@github.com>日期： 2020年12月12日周六 晚上8:57收件人： "paul-lee-attorney/fabric-2.1-gm" <fabric-2.1-gm@noreply.github.com>抄送： Subscribed <subscribed@noreply.github.com>主    题： Re: [paul-lee-attorney/fabric-2.1-gm] when i run the command "make", it entailed the error. (#1)
if you can not see the pic, the error info is etcdraft/store.go:19:2: module go.etcd.io/etcd@latest found (v3.3.25+incompatible), but does not contain package go.etcd.io/etcd/etcdserver/api/snap

**Comment by fantasticboy19** [comment]
how to solve the problem？
I set the etcd to the higher version, but it didn’t work either.

**Comment by paul-lee-attorney** [comment]
maybe can try to pull a lower version and store it into vendor folder or pkg folder?Regards.PaulFrom My Huawei Mate20-------- 原始邮件 --------发件人： immortal-fossil <notifications@github.com>日期： 2020年12月13日周日 上午9:58收件人： "paul-lee-attorney/fabric-2.1-gm" <fabric-2.1-gm@noreply.github.com>抄送： LI LI <paul_lee0919@163.com>, Comment <comment@noreply.github.com>主    题： Re: [paul-lee-attorney/fabric-2.1-gm] when i run the command "make", it entailed the error. (#1)

how to solve the problem？

I set the etcd to the higher version, but it didn’t work either.

**Comment by paul-lee-attorney** [comment]
I just tried unit test for BCCSP, while, the rest folder like MSP and INTERNAL are still left for further test. and, did not test network start up.  Regards.PaulFrom My Huawei Mate20-------- 原始邮件 --------发件人： immortal-fossil <notifications@github.com>日期： 2020年12月13日周日 上午9:58收件人： "paul-lee-attorney/fabric-2.1-gm" <fabric-2.1-gm@noreply.github.com>抄送： LI LI <paul_lee0919@163.com>, Comment <comment@noreply.github.com>主    题： Re: [paul-lee-attorney/fabric-2.1-gm] when i run the command "make", it entailed the error. (#1)

how to solve the problem？

I set the etcd to the higher version, but it didn’t work either.

**Comment by fantasticboy19** [comment]
thx , man

**Comment by paul-lee-attorney** [comment]
L。！99/！是I/.-------- 原始邮件 --------发件人： immortal-fossil <notifications@github.com>日期： 2020年12月13日周日 上午10:44收件人： "paul-lee-attorney/fabric-2.1-gm" <fabric-2.1-gm@noreply.github.com>抄送： LI LI <paul_lee0919@163.com>, Comment <comment@noreply.github.com>主    题： Re: [paul-lee-attorney/fabric-2.1-gm] when i run the command "make", it entailed the error. (#1)
thx , man

**Comment by fantasticboy19** [comment]
did the fabric-1.4-gm work well? or the fabric-1.4-gm does not work either.

**Comment by paul-lee-attorney** [comment]
current open version has some shortcomes, like not replace All hash funcs with SM3, and double hashing in SM2 signature algo ETC.  while, a friend might just release a New version. A friend right release a version, Pls try https://github.com/studyzy/fabric/tree/gmRegards.PaulFrom My Huawei Mate20-------- 原始邮件 --------发件人： immortal-fossil <notifications@github.com>日期： 2020年12月13日周日 下午4:56收件人： "paul-lee-attorney/fabric-2.1-gm" <fabric-2.1-gm@noreply.github.com>抄送： LI LI <paul_lee0919@163.com>, Comment <comment@noreply.github.com>主    题： Re: [paul-lee-attorney/fabric-2.1-gm] when i run the command "make", it entailed the error. (#1)
did the fabric-1.4-gm work well? or the fabric-1.4-gm does not work either.
